internal/namespaces/instance/v1/custom_security_group.go

```diff
@@ -243,6 +243,12 @@ func securityGroupClearCommand() *core.Command {
 		Resource:  "security-group",
 		Verb:      "clear",
 		ArgsType:  reflect.TypeOf(instanceResetSecurityGroupArgs{}),
+		Examples: []*core.Example{
+			{
+				Short:   "Remove all rules of the given security group",
+				Request: `{"security_group_id": "f8a5139c-6487-492d-a402-a9b9ffa21602"}`,
```

Comment:
All UUID used for examples should be 1111111-1111-1111-1111-111111111111
It makes it easier to notice this is an example and should be replaced to work properly.